Client/multiplayer_sa/CMultiplayerSA_WidescreenFix.cpp

```diff
@@ -0,0 +1,1111 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        multiplayer_sa/CMultiplayerSA_WidescreenFix.cpp
+ *
+ *  Multi Theft Auto is available from https://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+
+struct RsGlobal
```

Comment:
In the context of this PR, some fields are not used (frameLimit, quit, ps). In addition, this structure is incomplete relative to the SA code. If we do not need the remaining fields, use only those that are necessary.

Reply:
honestly, I've kept this because I was not sure if it would work without this fields, I took it off and it works